### ISSUE TITLE
Could you share the original code and datasets ?

### ISSUE THREAD

**Issue by ProQianXiao** [FGonzalezLopez/DeepSequencePPI#1]
Thanks for your work! 
I am interested in  prediction of PPI, your work  achieves a good effect.
The paper you mentioned  'DeepPPI: Boosting Prediction of Protein−Protein Interactions with
Deep Neural Networks' , is a great work.  But unfortunately, the link they provided is invalid now, I can not obtain their code and data. Could you share the original code and datasets if possible?
My email is proqianxiao@gmail.com. 

Thanks very much.

**Comment by FGonzalezLopez** [comment]
Thank you! 

It is indeed hard to find these datasets due to dead links. I'm afraid I can't provide you directly with the data, as I do not own them (hence the link to the source, instead of hosting them myself). That being said, I've been able to find a working link to some documents in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5085656/ (at the end of the abstract). These should contain the interaction data for at least S. cerevisiae core, H. pylori and H. sapiens.

I haven't looked into the data  they provide, but I'm guessing it's practically the same as those used in almost all the articles about PPI prediction. Good luck!

**Comment by ProQianXiao** [comment]
Thanks a lot, that's very kind of you!
The link you provide helps me a lot, because I was troubled with the original data and code mentioned in the paper .  By the way, I have another question, I run your code 30 times  and calculate the mean of the results, I got    Accuracy | Precision | Recall : 0.925997567 | 0.936395 | 0.914256,  precision is lower than 94.38%  in the paper,  does it mean that I should make some changes on the parameters?

**Comment by FGonzalezLopez** [comment]
No, that result is about what you would expect for the first dataset (the full S. cerevisiae data). The 94.38 probably refers to the 5-fold cross validation accuracy for the second dataset in the code, the S. cerevisiae core subset.

**Comment by ProQianXiao** [comment]
In the paper, the author writes 

>  To illustrate further the performance of DeepPPI, we compare DeepPPI with several traditional 
algorithms, including Nearest Neighbors, SVM, Decision Tree, Random Forest, AdaBoost, Naive Bayes,
Quadratic Discriminant Analysis (QDA). For S. cerevisiae data set, three-fourths of the data sets are used to training, one-fourth are used to evaluate these methods and compute all performance measures. Figure 5 shows the results of various methods on the S.cerevisiae.

![微信图片_20190521224803](https://user-images.githubusercontent.com/25978988/58106360-96a68480-7c1a-11e9-9102-a395060dc604.png)
                                                                                      Figure 5

I think the author used the full  S. cerevisiae data here, because he mentioned the core set of S. cerevisiae  with       '**S. cerevisiae Core Subset**'     in the later. But I am a little confused with Figure 5, the metrics in     '_Table 1. Comparison among Different Layer Architectures for DeepPPI on S. cerevisiae_'     is lower than        '_Performance comparison other algorithms on S. cerevisiae data set_'.

**Comment by FGonzalezLopez** [comment]
Well those differences are well within what you would expect for two different sets of experiments, the data is slightly different for each, so the results fluctuate (the deviation is also provided for most experiments). 

In fact, the results that we obtained, even though they were slightly better on average, were not found to be significantly different.

**Comment by ProQianXiao** [comment]
Yep, I understand now, thanks for your help. Your work helps me a lot.

**Comment by FGonzalezLopez** [comment]
Glad to hear that :)

I will be closing the issue in that case, good luck with your work.
